### ISSUE TITLE
4.9 Not Working With Magisk

### ISSUE THREAD

**Issue by zQueal** [AdroitAdorKhan/EnergizedProtection#61]
# Magisk Fully Updated and Working

![photo_2020-02-23_18-12-32](https://user-images.githubusercontent.com/239193/75122058-fd544500-5667-11ea-8921-795e1524f131.jpg)

# Energized Loads Fine at First

![photo_2020-02-23_18-12-26](https://user-images.githubusercontent.com/239193/75122068-11984200-5668-11ea-9a82-973082cf51c9.jpg)

# Fails & Hangs

![photo_2020-02-23_18-12-29](https://user-images.githubusercontent.com/239193/75122097-45736780-5668-11ea-979b-5c88ef4d27f2.jpg)

Magisk binary is `com.txQQ345io4oFgPML` which is randomized. Was working fine before, but maybe now it's not?

**Comment by AdroitAdorKhan** [comment]
Did you repackaged Magisk with random package name?

**Comment by AdroitAdorKhan** [comment]
Did you repackage Magisk with random package name?

**Comment by zQueal** [comment]
> Did you repackage Magisk with random package name?

Yes, but not recently. Several months ago. It's just not stopped working. Additionally, I've totally reinstalled Magisk without obfuscation and it's still not working.
